### DISCUSSION TITLE
We need a name!

### DISCUSSION THREAD

**Discussion by rungdung** [rungdung/Research-Narratives#46] [Q&A]
Research Narratives is a pretty long name (we think) and are on the lookout for names. We are also considering regional language roots (Kannada, Tamil, Malayalam, Telugu, etc) for the name. 

Edit to include suggestions that have been made previously 

Anglical:
- ResearchMapper
- NarrativeStudio
- MindLab
- Vizudio

Regional:
- ![Suggestions from Dheeraj](https://github.com/rungdung/Research-Narratives/assets/71073145/7847b431-c75a-4270-9e99-a3048a1abcf2)
- Prefix + Koota

Suggest away!

**Comment by soorajmr** [comment]
viztory
ಬರಹ (baraha) / bare / barahub
Iruve (ಇರುವೆ)  /  pipilika
ಕಾಗದ (kagada)
mapscribe / mapscript
euscribe / symscribe
plotlines / plottery
narratelab / narratehub
demoscribe
samvada
vizvrtta (visual vrittanta)
aakhyana
skrib / skrib lab / skrib studio
gignos / jignos / jignostudio
collegial / collegium 
synergia / synergium

**Reply by amruthkiran94** [reply]
- Mārgadarśi - Telugu (?)/Kannada - Guide
- Nakshe Kathe :) - _someone_ might have a problem with this 
- MapMyMind
- ReMap.story
- ReNaMap - "Research Narrative Map"
- ViReNa - "Visual Research Narrative"
- MindSmith
- ReadMyMap
- ChartsAreBoringMapsAreBetter

**Comment by rungdung** [comment]
Kagadha is my favorite as of now

**Comment by soorajmr** [comment]
Recursive acronyms is another option :)

shyns helps you narrate stories
stats together and tell stories
mapycs - a place you compose stories 
scods collaborate on data stories

**Comment by rungdung** [comment]
Consider the prefix 'Open' to emphasize on the nature of linking raw sources to sense making and output, making all of it available at the same time.
